src/components/graph-view.js

```diff
@@ -1033,7 +1039,8 @@ class GraphView extends React.Component<IGraphViewProps, IGraphViewState> {
             updatedNodes = selected?.nodes || new Map();
           }
 
-          return onUpdateNode(nodeMap.node, updatedNodes);
+          onUpdateNodePromise =
+            onUpdateNode(nodeMap.node, updatedNodes) || Promise.resolve();
```

Comment:
In onUpdateNode doesn't return a Promise then we'll assign a resolved one to the variable instead.